bors.toml

```diff
@@ -1,5 +1,5 @@
 status = [
-    "Clippy Test",
+    "Clippy/rustfmt Test",
```

Comment:
bors is waiting forever for this pull request since it checks if CI passes based on this file, which now uses an outdated name for the job.

Reply:
> bors is waiting forever for this pull request since it checks if CI passes based on this file, which now uses an outdated name for the job.

Brilliant!

Reply:
Hmmm bors.toml only lists basename and just "Test" would not be accurate, so I discarded that commit.  Oh well!  Maybe there's something smarter we can do some other time.

Thanks.

Reply:
Probably we should move clippy+rustfmt into ci.yml - I'm not sure there's any reason to separate them.  Again:  later.